README.md

```diff
@@ -39,33 +33,33 @@ Cursor/Claude Desktop → MCP Server → Contextual AI RAG Agent
 
 ## Installation
 
-
 1. Clone the repository:
 ```bash
 git clone https://github.com/ContextualAI/contextual-mcp-server.git
 cd contextual-mcp-server
 ```
 
-
 2. Create and activate a virtual environment:
 ```bash
 python -m venv .venv
 source .venv/bin/activate  # On Windows, use `.venv\Scripts\activate`
 ```
 
-
 3. Install dependencies:
 ```bash
 pip install -e .
 ```
 
-
 ## Configuration
 
+### Configure MCP Server
 
-### Environment Variables
+The server requires modifications of settings or use.
+For example, the single use server should be customized with an appropriate docstring for your RAG Agent.
 
-The server requires the following environment variables:
+The docstring for your query tool is critical as it helps the MCP client understand when to route questions to your RAG agent. Make it specific to your knowledge domain.
```

Comment:
Give an example of a "good" docstring

Reply:
added two examples, let me know if that helps